### ISSUE TITLE
Яндекс карты стали потреблять большое количество CPU

### ISSUE THREAD

**Issue by ks1322** [AdguardTeam/AdguardBrowserExtension#2508]
### AdGuard Extension version

4.2.168

### Browser version

Firefox 117.0

### OS version

Fedora 38

### What filters do you have enabled?

AdGuard Base filter, AdGuard Russian filter, AdGuard Annoyances filter, Fanboy's Annoyances, RU AdList - Counters

### Issue Details

https://yandex.ru/maps/
Яндекс карты стали долго загружаться при увеличении масштаба, перемещении карты. Браузер потребляет большое количество CPU. Такое ощущение что это началось после апдейта AdGuard Extension до версии 4.2.168. Можно ли как-нибудь откатиться на предыдущую версию?

### Expected Behavior

Яндекс карты быстро загружаются

### Screenshots

<details><summary>Screenshot 1:</summary>

</details>

**Comment by ks1322** [comment]
> Вы уверены, что это связано с расширением AdGuard?

Да, потому что проблема не проявляется после выключения (disable) AdGuard. После переустановки расширения проблема сохраняется.

**Comment by ks1322** [comment]
Looks like disabling "Send anonymous statistics of ad filters usage that will help us improve and optimize them" option improves performance significantly. Borrowed from here https://github.com/AdguardTeam/AdguardBrowserExtension/issues/2524#issuecomment-1753006199.

**Comment by windwak3r** [comment]
@ks1322 Is the issue still relevant?

**Comment by ks1322** [comment]
@windwak3r no, I can't see any difference with "Send anonymized extension usage data" option enabled or disabled in AdGuard Browser extension Version 5.2.700.1.
